streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -47,9 +46,9 @@ FhirConverter createFhirConverter() throws IOException {
 		String fhirBaseUrl = System.getProperty("openmrs.serverUrl") + System.getProperty("openmrs.fhirBaseEndpoint");
 		OpenmrsUtil openmrsUtil = new OpenmrsUtil(fhirBaseUrl, System.getProperty("openmrs.username"),
 		        System.getProperty("openmrs.password"), fhirContext);
-		FhirStoreUtil fhirStoreUtil = new FhirStoreUtil(System.getProperty("cloud.gcpFhirStore"), fhirContext);
-		IParser parser = fhirContext.newJsonParser();
-		return new FhirConverter(openmrsUtil, fhirStoreUtil, parser);
+		FhirStoreUtil fhirStoreUtil = new FhirStoreUtil(System.getProperty("cloud.gcpFhirStore"),
```

Comment:
This one should be conditional too; also change the property name from `cloud.gcpFhirStore` to something more generic.